scala-generator/src/main/scala/models/play/Play26Generator.scala

```diff
@@ -0,0 +1,26 @@
+package scala.models.play
+
+import io.apibuilder.generator.v0.models.{File, InvocationForm}
+import lib.generator.CodeGenerator
+
+object Play26Generator extends CodeGenerator {
+
+  override def invoke(form: InvocationForm): Either[Seq[String], Seq[File]] = {
+    val values = List(
+      files.Client(form),
```

Comment:
Is this the "full" client (incl models and json) or the client only? WOuld be good to distinguish between them, as in the http4s generator.

Reply:
Currently the full, this is where [the next PR](https://github.com/apicollective/apibuilder-generator/pull/491) keeps only the client.

It forces to download model, json, and client, but removes risks of conflicting definitions.